docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+:::
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Disable
+
+</xode-block>
+<xode-block title="Java">
+
+
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+disableProjection(projectionName: string, options?: DisableProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Disabling a projection that does not exists results in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#DisableNotFound
+
+</xode-block>
+<xode-block title="Java">
+
+
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+disableProjection(projectionName: string, options?: DisableProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+
+## Delete a projection
+
+Deletes a projection
+
+<xode-group>
+<xode-block title="C#">
+
+```csharp
+// This is currently not available in the .net client
+```
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+deleteProjection(projectionName: string, options?: DeleteProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Abort a projection
+
+Aborts a projection, this will not save the projection's checkpoint.
+
+:::warning
+The .net clients prior to version 21.6 had an incorrect behavior: they _will_ save the checkpoint. 
+:::
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Abort
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+// This is currently not directly available in the NodeJS client.
+// However, disableProjection allows to pass writeCheckpoint: false in the options in order to abort the projection  
+await client.disableProjection("projection to abort", { writeCheckpoint: false });
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Trying to abort a projection that does not exists will result in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Abort_NotFound
```

Comment:
Also, are other methods idempotent or only `Abort`?**CONCERN:** I think that part about calling abort multiple times would be good to put in the docs instead of a snippet. Also, are other methods idempotent or only `Abort`? If it's specific to the abort method, then it's worth keeping it as it is, but if it's in general, then it might be worth adding a general note or to the other specific methods.